### DISCUSSION TITLE
premium renewal problem

### DISCUSSION THREAD

**Discussion by drcagrituran25** [aclap-dev/video-downloadhelper#982] [Help and Q&A]
### Your question

Dear Sir/Madam,

I hope this message finds you well.

I have been a satisfied premium user of your program for the past 2-3 years. However, after a brief hiatus from using the software, I recently returned to find that despite having purchased a lifetime license, I am now being prompted to pay again. Unfortunately, I don't have any proof of payment. I don't know where to find it.

I would greatly appreciate your assistance in resolving this matter. As a loyal customer, I am keen to continue using the program without any interruptions.

Thank you for your understanding and support.

Best regards,
Çağrı TURAN

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: tr
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 1,
    "successfull_dl": 3
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

At the top of every page of this forum you can see a Wiki link.  Click that.  That takes you to what is the closest thing to a user manual that we have.  You should get in the habit of referring to it regularly.  The right side contains a table of contents.  Scrolling that down a bit you will see this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/c72607e0-cabc-40ec-a277-25553d7e8a4d)

Click the indicated chapter & follow those instructions.
